WooCommerce/Classes/ViewRelated/ReusableViews/LedgerTableViewCell.swift

```diff
@@ -106,7 +106,7 @@ private extension LedgerTableViewCell {
     enum Titles {
         static let subtotalLabel = NSLocalizedString("Product Total",
                                                      comment: "Product Total label for payment view")
-        static let feesLabel = NSLocalizedString("Fees",
+        static let feesLabel = NSLocalizedString("Custom amounts",
```

Comment:
```suggestion
        static let feesLabel = NSLocalizedString("Custom Amounts",
```

Nicer again for it to match the row above
![custom amounts capitalisation](https://github.com/woocommerce/woocommerce-ios/assets/2472348/f25fa1dd-5df3-4338-b36a-b321854f4f7b)